sdk/maps/azure-maps-search/README.md

```diff
@@ -0,0 +1,123 @@
+An **endpoint** and **credential** are necessary to instantiate the client object.
+
+
+## Looking up the endpoint
+
+You can find the endpoint for your Maps resource using the
+[Azure Portal][azure_portal_get_endpoint].
+
+> Note that the service requires a custom domain endpoint. Follow the instructions in the above link to format your endpoint:
+> https://{NAME-OF-YOUR-RESOURCE}.mapsservices.azure.com/
+
+
+## Create the client with AzureKeyCredential
+
+To use an [API key] as the `credential` parameter,
+pass the key as a string into an instance of [AzureKeyCredential][azure-key-credential].
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.maps.search import SearchClient
+
+endpoint = "https://<resource-name>.mapsservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+search_client = SearchClient(endpoint, credential)
+```
+
+## Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also
+authenticate with Azure Active Directory using the [azure-identity][azure_identity] library.
+
+To use the [DefaultAzureCredential][default_azure_credential] type shown below, or other credential types provided
+with the Azure SDK, please install the `azure-identity` package:
+
+```pip install azure-identity```
+
+You will also need to [register a new AAD application and grant access][register_aad_app] to your
+Translator resource by assigning the `"Cognitive Services User"` role to your service principal.
+
+Once completed, set the values of the client ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`.
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.maps.search import SearchClient
+credential = DefaultAzureCredential()
+
+search_client = SearchClient(
+    endpoint="https://<resource-name>.mapsservices.azure.com/",
+    credential=credential
+)
+```
+
+## Key concepts
+
+to add all operations
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common tasks, including:
+
+- [Operations](#events-operations)
+
+## Sample Code
+
+These are code samples that show common scenario operations with the Azure Maps Search client library.
+The async versions of the samples (the python sample files appended with `_async`) show asynchronous operations.
+Before run the sample code, refer to Prerequisites
+<!-- [Prerequisites](#Prerequisites) -->
+to create a resource, then set some Environment Variables
+
+```bash
+set AZURE_MAPS_SERVICE_ENDPOINT="https://<RESOURCE_NAME>.mapsservices.azure.com"
+set MAPS_SAMPLES_CONNECTION_STRING="<connection string of your Maps service>"
+
+pip install azure-maps-search
+
+python samples\search_client_sample.py
+python samples\search_client_sample_async.py
```

Comment:
Where is `samples` folder? Maybe `Sample code` and `Example` can be merged as one section.